jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KindFilenameMapperUtilTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
+
+import org.eclipse.jkube.kit.common.SystemMock;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KindFilenameMapperUtilTest {
+    private static final String MAPPING_PROPERTIES = "Pod=pd, pod";
+
+    private static final String VALID_TABLE = "cols=2*,options=\"header\"]" + System.lineSeparator()
+            + "|===" + System.lineSeparator()
```

Comment:
the `System.lineSeparator()` are making it hard to read. Are they really required?

Reply:
I'm really sorry for the delayed response, I have removed the line separators and used new line character, and removed the attached file